public/docs/_examples/tutorial/dart/pubspec.yaml

```diff
@@ -0,0 +1,15 @@
+name: angular2_tour_of_heroes
+version: 0.0.1
+environment:
+  sdk: '>=1.13.0 <2.0.0'
+dependencies:
+  angular2: 2.0.0-beta.1
+  browser: ^0.10.0
+  dart_to_js_script_rewriter: ^0.1.0+4
```

Comment:
I vote for including `dart_to_js_script_rewriter` in all tutorials. It also explained what this does in the quickstart.